chapter_appendix-mathematics-for-deep-learning/information-theory_vn.md

```diff
@@ -18,13 +18,13 @@ With information theory, we can measure and compare how much information is pres
 In this section, we will investigate the fundamental concepts of information theory and applications of information theory in machine learning.
 -->
 
-Chúng ta đang sống trong kỷ nguyên tràn ngập thông tin.
+Chúng ta đang sống trong kỷ nguyên tràn ngập thông tin. 
 Thông tin cung cấp một ngôn ngữ chung cho các lĩnh vực: từ thơ của Shakespeare đến các bài báo khoa học của các nhà nghiên cứu trên Cornell ArXiv,
```

Comment:
```suggestion
Thông tin cung cấp một ngôn ngữ chung vượt qua những vực thẳm phân tách nhiều lĩnh vực riêng biệt: từ thơ của Shakespeare đến các bài báo khoa học của các nhà nghiên cứu trên Cornell ArXiv,
```

Reply:
Câu này nghe complicated quá lkphuc ạ :).  Chữ 'vực thẳm phân tách' có lẽ nên sửa là ra 'hàng rào ngăn cách' cho nó nhẹ nhàng hơn chăng?

Reply:
Tại bên tiếng anh có [hình ảnh](https://cdn.disclose.tv/sites/default/files/styles/og_image/public/img/post/2016/12/03/chasm-of-ignorance-physics-genius-reveals-how-everything-is-connected-136835.jpg?itok=12zpFuAJ) `chasm of ignorance` nên em dịch sát. Dịch thân quen với tiếng việt cũng được tuy bị mất đi hình ảnh này:
```suggestion
Thông tin cung cấp một ngôn ngữ chung giúp vượt qua rào cản ngăn cách nhiều lĩnh vực riêng biệt: từ thơ của Shakespeare đến các bài báo khoa học của các nhà nghiên cứu trên Cornell ArXiv,
```